build/selinux/simp-environment.fc

```diff
@@ -1 +1,27 @@
-/var/simp(/.*)?     system_u:object_r:puppet_var_lib_t:s0
+#IN order for puppetserver to access files it needs a puppet_var_lib_t context.
+/var/simp(/.*)?                                                                                                     system_u:object_r:puppet_var_lib_t:s0
+/var/simp/(?!environments)[^/]+(/.*)?                                                                               system_u:object_r:puppet_var_lib_t:s0
+/var/simp/environments                                                                                              system_u:object_r:puppet_var_lib_t:s0
+/var/simp/environments/[^/]+                                                                                        system_u:object_r:puppet_var_lib_t:s0
+#Anything in an environment not under rsync should have puppet server access
+/var/simp/environments/[^/]+/(?!rsync)[^/]+(/.*)?                                                                   system_u:object_r:puppet_var_lib_t:s0
```

Comment:
Doesn't the first rule do this?

It seems like `/var/simp(/.*)?` would cover anything under `/var/simp/*` at any level